packages/mui-material/src/styles/createTheme.spec.ts

```diff
@@ -218,6 +218,7 @@ const theme = createTheme();
             variants: [
               {
                 props: ({ ownerState }) => ownerState.color === 'primary',
+                style: {},
```

Comment:
Wasn't style required before?

Reply:
It was, but the `CSSObjectWithVariants` wasn't being picked correctly, so it wasn't enforced. Is that correct @siriwatknp?

Reply:
Yes. that's correct.